lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -162,7 +162,8 @@ defmodule ExUnit.Callbacks do
     quote bind_quoted: [var: escape(var), block: escape(block)] do
       name = :"__ex_unit_setup_#{length(@ex_unit_setup)}"
       defp unquote(name)(unquote(var)), unquote(block)
-      @ex_unit_setup [{name, @ex_unit_describe} | @ex_unit_setup]
```

Comment:
Instead of doing the unmatching here, keep this change and the one below exactly as is and instead change [this line](https://github.com/arnodirlam/elixir/blob/3598d65ff0f1681bb5047672b57023ceda4a5804/lib/ex_unit/lib/ex_unit/callbacks.ex#L413) to be like this:

    defp compile_merge({callback, {_line, describe}}) do

```elixir

Reply:
Okay great! Thanks for taking the time to guide me to the best way to do this. Is there anything else that could be improved?